buildpacks/kotlin-gradle/README.md

```diff
@@ -0,0 +1,12 @@
+# Sample: Kotlin Gradle Buildpack
+
+Compatible apps:
+- Kotlin apps that use Gradle
+
+### Usage
+
+```bash
+pack build sample-app --builder <builder> --buildpack . --path ../../apps/kotlin-gradle 
```

Comment:
i'm not sure if we want to recommend this here (an arbitrary builder won't just work unless the buildpack is compatible with its stack). Maybe we should just point them to the main docs?

Reply:
As discussed, I think we might go with pointing to a sample-builder to guarantee compatibility and either make it a prerequisite or have it available in docker hub.